lib/python/grassdb/checks.py

```diff
@@ -134,6 +139,46 @@ def get_lockfile_if_present(database, location, mapset):
     return None
 
 
+def get_datacatalog_notification_items(title):
```

Comment:
This should probably go into the manager. This is specific to GUI, so it shouldn't be here.